### PR TITLE
Feature/docker secrets

### PR DESCRIPTION
**Pull request by RobPiwowarek**

### CODE REVIEW

**Comment by RobPiwowarek** [comment]
Added 2 placeholder secrets through repository settings pubk privk

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ jobs:
           export CONTAINER_LOWERCASE="${CONTAINER_IMAGE,,}"
           echo $CONTAINER_LOWERCASE
           docker login -u $GITHUB_ACTOR -p ${{secrets.GITHUB_TOKEN}} docker.pkg.github.com
-          docker build -t $CONTAINER_LOWERCASE .
+          docker build --build-arg PUBLIC_KEY=${{secrets.pubk}} --build-arg PRIVATE_KEY=${{secrets.privk}}-t $CONTAINER_LOWERCASE .
```
Comment (FrozenPear42) [inline .github/workflows/publish.yml]: Missing space before `-t`.
You also have to modify  Docker file save those args to file and then copy it with exisitng `COPY` commands

**Review by FrozenPear42 — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (RobPiwowarek) [reply .github/workflows/publish.yml]: so the existing COPY which copies dockerfile public/private key wont work?

**Review by FrozenPear42 — APPROVED** [review]
LGTM
ap,550x550,12x16,1,transparent,t u7

**Pull request merged**